resources/helm/v2.0/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -282,6 +285,10 @@ spec:
           - name: ISTIO_META_MESH_ID
             value: "{{ $.Values.global.trustDomain }}"
           {{- end }}
+          {{- if not $gateway.runAsRoot }}
```

Comment:
Is it being applied twice?

Reply:
No, this change is here because this is a backport of my upstream fix. We're not setting `$gateway.runAsRoot` but are instead using `sed` to make the necessary changes

Reply:
ie this change comes is from the `maistra/istio` repo, I'm just pulling it in here

Reply:
ah :)

Reply:
I approved this. It looks fine to me. Though, I wonder why we don't set `$gateway.runAsRoot==false`.